src/library/modules/ImageExport.js

```diff
@@ -256,6 +256,25 @@
     return link;
   };
 
+  KC3ImageExport.deleteUpload = function (deletehash) {
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: 'https://api.imgur.com/3/image/' + deletehash,
+        method: 'DELETE',
+        headers: {
+          Authorization: 'Client-ID 088cfe6034340b1',
+          Accept: 'application/json',
+        },
+        processData: false,
+        contentType: false,
```

Comment:
Should be able to remove `processData: false,` and `contentType: false`; they were only necessary to tell jQuery not to mangle the Blob upload.